content/zh/docs/reference/config-api/kubeadm-config.v1beta3.md

```diff
@@ -0,0 +1,1751 @@
+use it to customize the node name, the CRI socket to use or any other settings that should apply to this
+node only (e.g. the node ip).</p>
+</li>
+<li>
+<p>LocalAPIEndpoint, that represents the endpoint of the instance of the API server to be deployed on this node;
+use it e.g. to customize the API server advertise address.</p>
+-->
+<p>类型 InitConfiguration 用来配置运行时设置，就 kubeadm init 命令而言，包括
+启动引导令牌以及所有与 kubeadm 所在节点相关的设置，包括：</p>
+
+<ul>
+<li>nodeRegistration：其中包含与向集群注册新节点相关的字段；使用这个类型来
+定制节点名称、要使用的 CRI 套接字或者其他仅对当前节点起作用的设置
+（例如节点 IP 地址）。</li>
+<li>localAPIEndpoint：代表的是要部署到此节点上的 API 服务器示例的端点；
+使用这个类型可以完成定制 API 服务器公告地址这类操作。</li>
+</ul>
+
+<pre><code>apiVersion: kubeadm.k8s.io/v1beta3
+kind: ClusterConfiguration
+networking:
+  ...
+etcd:
+  ...
+apiServer:
+  extraArgs:
+    ...
+  extraVolumes:
+    ...
+...
+</code></pre>
+
+<!--
+<p>The ClusterConfiguration type should be used to configure cluster-wide settings,
+including settings for:</p>
+<ul>
+<li>
+<p>Networking, that holds configuration for the networking topology of the cluster; use it e.g. to customize
+Pod subnet or services subnet.</p>
+</li>
+-->
+<p>类型 `ClusterConfiguration` 用来定制集群范围的设置，具体包括以下设置：</p>
+
+<ul>
+<li><code>networking</code>：其中包含集群的网络拓扑配置。使用这一部分可以定制 Pod 的
+子网或者 Service 的子网。</li>
+<!--
+<li>
+<p>Etcd configurations; use it e.g. to customize the local etcd or to configure the API server
+for using an external etcd cluster.</p>
+</li>
+<li>
+<p>kube-apiserver, kube-scheduler, kube-controller-manager configurations; use it to customize control-plane
+components by adding customized setting or overriding kubeadm default settings.</p>
+-->
+<li><code>etcd</code>：etcd 数据库的配置。例如使用这个部分可以定制本地 etcd 或者配置 API 服务器
+使用一个外部的 etcd 集群。</li>
+<li><code>kube-apiserver</code>、<code>kube-scheduler</code>、<code>kube-controller-manager</code>
+配置：这些部分可以通过添加定制的设置或者重载 kubeadm 的默认设置来定制控制面组件。</li>
+</ul>
+
+<pre style="background-color:#fff"><span style="color:#000;font-weight:bold">apiVersion</span>:<span style="color:#bbb"> </span>kubeproxy.config.k8s.io/v1alpha1<span style="color:#bbb">
+</span><span style="color:#bbb"></span><span style="color:#000;font-weight:bold">kind</span>:<span style="color:#bbb"> </span>KubeProxyConfiguration<span style="color:#bbb">
+</span><span style="color:#bbb">  </span>...<span style="color:#bbb">
+</span></pre>
+
+<!--
+<p>The KubeProxyConfiguration type should be used to change the configuration passed to kube-proxy instances
+deployed in the cluster. If this object is not provided or provided only partially, kubeadm applies defaults.</p>
+<p>See https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/ or
+https://godoc.org/k8s.io/kube-proxy/config/v1alpha1#KubeProxyConfiguration
+for kube-proxy official documentation.</p>
+-->
+<p>KubeProxyConfiguration 类型用来更改传递给在集群中部署的 kube-proxy 实例
+的配置。如果此对象没有提供，或者仅部分提供，kubeadm 使用默认值。</p>
+
+<p>关于 kube-proxy 的官方文档，可参阅
+https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/
```

Comment:
```suggestion
https://kubernetes.io/zh/docs/reference/command-line-tools-reference/kube-proxy/
```

I think it would be better to connect to a Chinese document. ^_^

Reply:
good suggestion.